apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml

```diff
@@ -58,6 +58,10 @@ spec:
                         value: dcdcftappsstgkv
                       - key: BUSINESS_AREA_TAG
                         value: CFT
+                      - key: PROD_AKS_CLUSTER_NAME
```

Comment:
is this actually used? jenkins shouldn't be talking to prod AKS

Reply:
@ciaranca as discussed, we dont need this for Production

Reply:
Thanks guys.